src/main/java/bowlingRefactor/Main.java

```diff
@@ -0,0 +1,27 @@
+package bowlingRefactor;
+
+import bowlingRefactor.domain.FrameNumber;
+import bowlingRefactor.domain.Frames;
+import bowlingRefactor.domain.Player;
+import bowlingRefactor.view.InputView;
+import bowlingRefactor.view.ResultView;
+
+public class Main {
+    public static void main(String[] args) {
+        String bowlerName = InputView.inputUserNames();
+        Player bowler = new Player(bowlerName);
+
+        ResultView.printScoreBoard(bowler);
+        for (int i = 0; i < FrameNumber.MAX_FRAME_NUMBER; i++) {
+            play(bowler, i);
+        }
+    }
+
+    private static void play(Player bowler, int index) {
+        while (bowler.isLeftPin(index)) {
+            int pin = InputView.inputPinCount(bowler.get(index).getFrameNumber());
```

Comment:
> 규칙 4: 한 줄에 점을 하나만 찍는다.

규칙4를 고민하시다 보면 자연스럽게 객체 안에 필요한 기능을 만드시게 될거에요. 🙂 

```suggestion
            FrameNumber frameNumber = bowler.getFrameNumberOf(index);
```
```java
//class Player
public FrameNumber getFrameNumberOf(int index) {
    Frame frame = frames.get(index);
    return frame.getFrameNumber();
}
```